pipeline.tpl.yml

```
@@ -171,6 +171,11 @@ resources:
   source:
     branch: master
     uri: https://((github-token))@github.com/concourse/charts
+    paths:
+    - stable/concourse/
+    - stable/postgres/
+    - stable/grafana/
+    - stable/prometheus/
 
```

```suggestion
    depth: 5
    
```
Add `depth` so we avoid pulling in the entire history of the helm/charts repo.

---

I think depth 5 should be OK. If not jump to a higher number (say 1024) and then use a binary search to pick a number (try 512, 256, 128 ...).

---

sounds good!